sal/class.json-api-site-jetpack.php

```diff
@@ -188,12 +188,17 @@ function current_user_can( $role ) {
 	}
 
 	function is_fse_active() {
-		$fse_enabled = Jetpack::is_plugin_active( 'full-site-editing/full-site-editing-plugin.php' );
-		$has_method  = method_exists( '\A8C\FSE\Full_Site_Editing', 'is_supported_theme' );
-		if ( $fse_enabled && $has_method ) {
-			$fse  = \A8C\FSE\Full_Site_Editing::get_instance();
-			$slug = get_option( 'stylesheet' );
-			return $fse->is_supported_theme( $slug );
+		if ( ! Jetpack::is_plugin_active( 'full-site-editing/full-site-editing-plugin.php' ) ) {
+			return false;
+		}
+		if ( apply_filters( 'a8c_disable_full_site_editing', false ) ) {
```

Comment:
I'd be happy with a docblock here to keep our Codex parser happy. It would also help explain what the filter is going to be used for, as I think it may be a bit confusing for non-a12s right now.

Here is an example:
https://github.com/Automattic/jetpack/blob/b999369daf5a37988f1d9e4419ac128ac59219d6/sal/class.json-api-site-base.php#L115-L124

Reply:
Will do, thanks!